PyOpenWorm/dataObject.py

```diff
@@ -445,12 +449,10 @@ def __getitem__(self, x):
                 (x, self))
 
     def get_owners(self, property_class_name):
-        """ Return the owners along a property pointing to this object """
-        res = []
+        """ Return a generator of owners along a property pointing to this object """
```

Comment:
Just curious why you want this to be a generator -- there's no other code change related to it.

Reply:
I **think** it's a good practice to return generator and not list. And as this method is never used, so it's easy to change. But maybe it's not the good way here ?

Reply:
No, I agree. I prefer not to create a list if it's not needed, just thought you were contemplating an immediate use for this method.